internal/controller/providerconfig/healthcheck_controller.go

```diff
@@ -254,3 +268,28 @@ func (r *HealthCheckReconciler) setupWithManager(mgr ctrl.Manager) error {
 		}.ForControllerRuntime()).
 		Complete(r)
 }
+
+func (r *HealthCheckReconciler) unpauseBuckets(ctx context.Context, s3BackendName string) error {
+	buckets := &v1alpha1.BucketList{}
+	matchingLabels := client.MatchingLabels{
+		s3BackendName: "true",
```

Comment:
As i understand true doesn't mean anything. Kubernetes supports empty labels. Does it make sense to use empty label instead?

Reply:
Nice, I didn't know that k8s supports empty labels 😄